packages/site/cms-site/src/blocks/DamVideoBlock.tsx

```diff
@@ -33,6 +46,13 @@ export const DamVideoBlock = withPreview(
         const [showPreviewImage, setShowPreviewImage] = useState(true);
         const hasPreviewImage = Boolean(previewImage && previewImage.damFile);
 
+        const inViewRef = useRef<HTMLDivElement>(null);
```

Comment:
Do you need the extra inViewRef? Wouldn't it be possible to use the videoRef to check if it's in view?

Reply:
I already tried that, but using only one ref is not possible, as controlling the videos does not work in that case.